index.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var bin = require('./lib');
+var path = require('path');
 
 /**
  * Module exports
  */
 
-module.exports.path = bin.path();
```

Comment:
I'd rather see `bin.path()` being lightweight instead. Read, not require any dependencies.